20181204-Cloud/pastpaper 2015.java

```diff
@@ -10,7 +10,7 @@ public String getHint(String a){
     for(int i=0;i<a.length();i++){
       if((puzzle.substring(i,i+1)).equals.(a.substring(i,i+1)))
       {hidden+=puzzle.substring(i,i+1));}
-      else(if(!puzzle.indexOf(a.substring(i,i+1)))
+      else(if(puzzle.indexOf(a.substring(i,i+1)==-1))
            {hidden+="*";}
            else
            {hidden+="+";}
```

Comment:
**Line 13**
else(if(puzzle.indexOf(a.substring(i,i+1)==-1)) should be 
else if(puzzle.indexOf(a.substing(i, i + 1) == 1)

**Line 19**
There should be more one "}"